default_app/main.js

```diff
@@ -253,13 +253,16 @@ function loadApplicationPackage (packagePath) {
   } catch (e) {
     if (e.code === 'MODULE_NOT_FOUND') {
       app.focus()
-      dialog.showErrorBox(
-        'Error opening app',
-        'The app provided is not a valid Electron app, please read the docs on how to write one:\n' +
-        `https://github.com/electron/electron/tree/v${process.versions.electron}/docs
-
-${e.toString()}`
-      )
+      dialog.showMessageBox({
+        message: 'Error opening app',
+        detail: 'Unable to open or find an Electron app. Click to learn more on how to write an app.\n\n' +
+            `${e.toString()}`,
+        buttons: ['Learn More', 'OK']
```

Comment:
I think you'll want to have to OK button be the default selected button instead of Learn More, seems like a more common pattern since the person might know what to do just by seeing the error message and might not want to always learn more.